### ISSUE TITLE
rowEditingStopped not firing on V28.1

### ISSUE THREAD

**Issue by awdorrin** [ag-grid/ag-grid#5464]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```
With editType=fullRow the rowEditingStopped event is not firing, although rowEditingStarted is firing.
Reverted back to 28.0.0 and the rowEdtingStopped event started firing again

**Please tell us about your environment:**
    "@angular/core": "^14.1.1",
    "@angular/material": "^14.1.1",
    "ag-grid-angular": "^28.1.0",
    "ag-grid-community": "^28.1.0",

```
        <ag-grid-angular #agGridLinks
                           class="ag-theme-balham app-aggrid"
                           [columnDefs]="linksColDefs"
                           [gridOptions]="linksGridOpts"
                           (gridReady)="linksOnGridReady($event)"
                           (firstDataRendered)="linksOnFirstDataRendered()"
                           (cellEditRequest)="linksOnCellEditRequest($event)"
                           (rowEditingStarted)="linksOnRowEditingStarted($event)"
                           (rowEditingStopped)="linksOnRowEditingStopped($event)"
                           [rowData]="linksData">
          </ag-grid-angular>
```

```
    this.linksGridOpts = {
      multiSortKey: 'ctrl',
      pagination: true,
      paginationAutoPageSize: true,
      context: this,
      editType: 'fullRow',
      readOnlyEdit: true,
      suppressClickEdit: true,
      stopEditingWhenCellsLoseFocus: false,
```

**Comment by awdorrin** [comment]
This may be fixed by: AG-7118 [Full Row Edit] Cannot Edit the same row twice

**Comment by awdorrin** [comment]
Thanks, I have confirmed that the fix in AG-7118 does resolve my issue. Thanks for the feedback

**Comment by jnre** [comment]
hi i had the same issue in 28.1, i had to revert to 27.3 for it to work for react.

**Comment by StephenCooper** [comment]
We have just released a patch version of `28.1.1` which now resolves the issues with Full Row Editing.
